content/en/docs/armory-admin/caching-agents.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Configuring clouddriver caching agents
+linkTitle: Configuring clouddriver caching agents
+aliases:
+  - /docs/spinnaker-install-admin-guides/caching-agents/
+---
+
+Clouddriver is the Spinnaker service responsible for caching cloud infrastructure like EC2 instances, kubernetes pods, docker images, etc. It does it by running caching agents in separate threads at scheduled intervals that query the infrastructure and save the result in redis or a sql datastore.
+
+Depending on the number of elements of the infrastructure being cached, you would require to increase the number of replicas of clouddriver and/or increase its cpu and memory limits.
+
+The following configuration settings affect the behavior of the caching agents and can be used to adjust them depending on the size of the infrastructure being cached. If using the Spinnaker Operator, these settings live under the key `.spec.spinnakerConfig.profiles.clouddriver` of `SpinnakerService` manifest. If using Halyard, they live in `.hal/<profile name>/profiles/clouddriver-local.yml`.
+
+### SQL global caching agents configuration
+
+|Key|Description|Default Value|
+|----|---|---|
+|sql.scheduler.enabled|Enable or disable the sql scheduler|false
```

Comment:
Wrap options in backticks? That's the style I used for agent options so it'd be nice to be consistent.